lib/relay.js

```diff
@@ -112,6 +114,9 @@ function responseHandler(filterRules, config) {
         headers: headers,
         method,
         body,
+        agentOptions: {
+          ca: (() => {return config.caCert ? fs.readFileSync(path.resolve(process.cwd(), config.caCert)) : undefined;})(),
```

Comment:
This is going to read the config file on every request right? I think we should just read it once in `/lib/config/index.js`, and set `config.caCert` to the contents of the file. A CA cert is small enough that the memory overhead doesn't matter. What do you think?